examples/c/README.md

```diff
@@ -34,6 +34,9 @@ For other options to download models, read through [our download options](https:
       - CPU / QNN execution provider: `onnxruntime-genai-<version>-<platform>.zip/tar.gz`
       - CUDA execution provider: `onnxruntime-genai-<version>-<platform>-cuda.zip/tar.gz`
       - DirectML execution provider: `onnxruntime-genai-<version>-<platform>-dml.zip/tar.gz`
+      - TRT-RTX execution provider: `onnxruntime-genai-trt-rtx-win-x64-<version>.zip` 
+        - **Note**: TRT-RTX packages are distributed separately from [NVIDIA's onnxruntime-genai repository](https://github.com/NVIDIA/onnxruntime-genai/releases)
+        - Download [onnxruntime-genai-trt-rtx-win-x64-0.10.0.zip](https://github.com/NVIDIA/onnxruntime-genai/releases/download/v0.10.0/onnxruntime-genai-trt-rtx-win-x64-0.10.0.zip) for v0.10.0 
```

Comment:
Why would trt ep need a separate c package? Should it not be identical to the cuda one?